addons/website_sale/models/product.py

```diff
@@ -200,7 +200,7 @@ def _website_price(self):
         ret = self.env.user.has_group('sale.group_show_price_subtotal') and 'total_excluded' or 'total_included'
 
         for p, p2 in zip(self, self2):
-            taxes = partner.property_account_position_id.map_tax(p.taxes_id).filtered(lambda x: x.company_id == company_id)
+            taxes = partner.property_account_position_id.map_tax(p.sudo().taxes_id.filtered(lambda x: x.company_id == company_id))
```

Comment:
OK, but let's try to refocus first this PR. So are you saying that without multi-company, you're having permission issues in this code? If so, then the solution is to add the corresponding ACLs/record rules for accessing the involved objects. Can you put the steps to reproduce the problem in runbot?